core/vm/allow_list.go

```diff
@@ -17,9 +17,38 @@ const (
 
 type allowListPrecompile struct{}
 
+var (
+	enabledKey = common.Hash{}
```

Comment:
TODO: is setting a key of `0x000` the same as deleting something?

Reply:
nit: `statusKey`

Reply:
Yes: https://github.com/ava-labs/subnet-evm/blob/master/core/state/state_object.go#L363

Reply:
That link refers to the value, not the key itself.